monte-carlo/qiskit/mc_benchmark.py

```diff
@@ -505,7 +505,7 @@ def get_args():
     parser.add_argument("--skip_qubits", "-k", default=1, help="Number of qubits to skip", type=int)
     parser.add_argument("--max_circuits", "-c", default=3, help="Maximum circuit repetitions", type=int)  
     parser.add_argument("--method", "-m", default=1, help="Algorithm Method", type=int)
-    parser.add_argument("--num_state_qubits", "-nsq", default=0.0, help="Number of State Qubits", type=int)
```

Comment:
`0.0` is wrong due to the definition of `int`.

Reply:
Would it make sense to change the default value to `1` since the `MIN_STATE_QUBITS` variable in the script is set to be `1`?

Reply:
The current default value 0 causes an error. The default value should be valid, but I'm fine with any value rather than 0.